first_analysis.ipynb

```diff
@@ -485,7 +487,10 @@
     "## Bonus exercise\n",
     "\n",
     "Have a look at the Pandas package especially the `read_csv` method. Could you\n",
-    "solve the previous exercise using this functionality?"
+    "solve the previous exercise using this functionality?\n",
+    "\n",
+    "You'll need to install pandas first, which you can easily do by executing\n",
+    "`!pip install "pandas==1.3"` within a new cell."
```

Comment:
Why `pandas == 1.3`?

Reply:
Just to say - I checked with a Python 3.7 virtualenv - `pip install pandas` does give version 1.3.

Reply:
Sure, no problem with taking the pinning away.